cmd/infra/powervs/create.go

```diff
@@ -1114,7 +1114,7 @@ func (infra *Infra) setupPowerVSDHCP(options *CreateInfraOptions, session *ibmpi
 // createPowerVSDhcp creates a new dhcp server in powervs
 func (infra *Infra) createPowerVSDhcp(options *CreateInfraOptions, client *instance.IBMPIDhcpClient) (dhcpServer *models.DHCPServerDetail, err error) {
 	startTime := time.Now()
-	dhcp, err := client.Create(&models.DHCPServerCreate{CloudConnectionID: &infra.PowerVSCloudConnectionID})
+	dhcp, err := client.Create(&models.DHCPServerCreate{CloudConnectionID: &infra.PowerVSCloudConnectionID, DNSServer: utilpointer.String("1.1.1.1")})
```

Comment:
can you provide some context and add a comment on why is this needed and where is it coming from?

Reply:
Added a comment.